JASP-Engine/JASP/R/commonMachineLearningRegression.R

```diff
@@ -302,7 +302,7 @@
 
   regressionResult <- jaspResults[["regressionResult"]]$object
   
-  predPerformance <- data.frame(true = regressionResult[["testReal"]], predicted = regressionResult[["testPred"]])
+  predPerformance <- data.frame(true = c(regressionResult[["testReal"]]), predicted = regressionResult[["testPred"]])
```

Comment:
Is this on purpose?

Reply:
Yes, it is required AND on purpose!
```r
a <- matrix(1)
colnames(a) <- "hoi"
> data.frame(true = a)
  hoi
1   1
> data.frame(true = c(a))
  true
1    1
```
the next bit in the code is a ggplot with `aes(... = true)` so the version without `c(...)` would have the wrong names.